networkx/algorithms/covering.py

```diff
@@ -55,6 +55,11 @@ def min_edge_cover(G, matching_algorithm=None):
 
     Minimum edge cover for bipartite graph can also be found using the
     function present in :mod:`networkx.algorithms.bipartite.covering`
+
+    Examples
+    --------
+    >>> G = nx.Graph([(0, 1), (1, 2), (1, 5), (5, 4), (2, 4), (2, 3), (4, 3)])
+    >>> edge_cover = nx.min_edge_cover(G)
```

Comment:
It'd be useful for the reader to be able to see the actual output of the function. For example, you could not bind the output to a variable:

```
>>> nx.min_edge_cover(G)
{(1, 0), (3, 2), (4, 5)}
```

or you can show the result by putting it on it's own line in the example (just like you would in a Python interpreter). This can be useful if you want to use the result in a subsequent example to further illustrate a point, as you've done in the `is_edge_cover` example.

Reply:
Thank you very much, sir, I will adjust it.